modules/must-gather-flags.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="must-gather-flags_{context}"]
+= Flags available to use with the must-gather command
```

Comment:
I'm not sure if it's okay to capitalize, but if possible I'd recommend making this more concise, maybe 

```suggestion
= Must-gather flags
```